judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl

```diff
@@ -89,7 +89,8 @@ rule CreateMappedTransferAttributeEntityDefault
         and s.getterExpr.features.first.member.defaultExpr.isDefined() and not JSL!TransferDefault.all().exists(d | d.member.reference == s)
       t.name = defaultDefaultNamePrefix + s.name + defaultDefaultNameMidfix + s.getterExpr.features.first.member.eContainer.name + defaultDefaultNamePostfix;
       t.setId("(jsl/" + s.getId() + ")/CreateMappedTransferAttributeEntityDefault");      
-      t.binding = s.getterExpr.features.first.member.defaultExpr.equivalent("CreateDefaultValueForPrimitiveEntityMember");               
+      t.binding = s.getterExpr.features.first.member.defaultExpr.equivalent("CreateDefaultValueForPrimitiveEntityMember");
+      t.required = false;
```

Comment:
This seems to override the original behaviour in `AbstractCreateTransferAttributeForTransferFieldDeclaration` which is `t.required = s.isRequired`. Is it really the intended solution? Wouldn't a proper solution for [JNG-4942](https://blackbelt.atlassian.net/browse/JNG-4942) - by modifying validation procedures - could solve this as well as the other cases?

Reply:
I agree

Reply:
I agree

Reply:
I agree.

Reply:
[49e7d9d](https://github.com/BlackBeltTechnology/judo-tatami-jsl/pull/187/commits/49e7d9d7a81aaeb67d04d9a399e7f2533f949213)